app/src/main/java/comcmput301f16t01/github/carrier/HelpActivity.java

```diff
@@ -0,0 +1,13 @@
+package comcmput301f16t01.github.carrier;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class HelpActivity extends AppCompatActivity {
```

Comment:
I do in general like the way HelpActivity looks. It seems a little bit empty, try spreading out the spacing between the icons in the legend, that might help. This screen is a little harsh on the eyes with all the white but we'll talk about colour/icons later.